@kiva/kv-components/vue/KvCarousel.vue

```diff
@@ -29,7 +29,7 @@
 		<!-- Carousel Controls -->
 		<div
 			v-if="slideIndicatorCount > 1"
-			class="tw-flex
+			class="kv-carousel__controls tw-flex
```

Comment:
Are these two added classes defined somewhere else?

Reply:
I added these so I could reposition the controls in this pr https://github.com/kiva/ui/pull/4169